src/index.tsx

```diff
@@ -619,11 +634,20 @@ class App extends React.Component<{}, AppState> {
       >
         <fieldset>
           <legend>Shapes</legend>
-          {this.renderOption({ type: "rectangle", children: "Rectangle" })}
-          {this.renderOption({ type: "ellipse", children: "Ellipse" })}
-          {this.renderOption({ type: "arrow", children: "Arrow" })}
-          {this.renderOption({ type: "text", children: "Text" })}
-          {this.renderOption({ type: "selection", children: "Selection" })}
+          {SHAPES.map(({ value, label }) => (
+            <label>
+              <input
+                type="radio"
+                checked={this.state.elementType === value}
+                onChange={() => {
+                  this.setState({ elementType: value });
+                  clearSelection();
+                  this.forceUpdate();
+                }}
+              />
+              <span>{label}</span>
```

Comment:
Would be cool to underline the letter that's the shortcut like on Microsoft 95 :)
https://twitter.com/tuomassalo/status/978717292023500805